test/e2e/catalog_e2e_test.go

```diff
@@ -272,6 +269,15 @@ var _ = Describe("Catalog represents a store of bundles which OLM can use to ins
 		})
 		Expect(err).ShouldNot(HaveOccurred())
 
+		Eventually(func() types.UID {
+			// Get updated configmap
+			updatedConfigMap, err := c.KubernetesInterface().CoreV1().ConfigMaps(testNamespace).Get(context.Background(), fetchedInitialCatalog.Spec.ConfigMap, metav1.GetOptions{})
+			Expect(err).ShouldNot(HaveOccurred())
+			return updatedConfigMap.ObjectMeta.OwnerReferences[0].UID
+		}).Should(Equal(fetchedUpdatedCatalog.ObjectMeta.UID))
+		updatedConfigMap, err := c.KubernetesInterface().CoreV1().ConfigMaps(testNamespace).Get(context.Background(), fetchedInitialCatalog.Spec.ConfigMap, metav1.GetOptions{})
```

Comment:
Nit, non-blocking: Can we add a space here so it is easier to read?

```suggestion
		}).Should(Equal(fetchedUpdatedCatalog.ObjectMeta.UID))
		
		updatedConfigMap, err := c.KubernetesInterface().CoreV1().ConfigMaps(testNamespace).Get(context.Background(), fetchedInitialCatalog.Spec.ConfigMap, metav1.GetOptions{})
```

Reply:
Thanks for review.  I add the space.